config/config.go

```diff
@@ -124,6 +124,10 @@ type Proxy struct {
 
 	// BufferSize represent the reverse proxy buffer size.
 	BufferSize uint64 `yaml:"buffer_size"`
+
+	// BypassURLPath represents URL path that requires bypassing authorization.
+	// Setting this configuration may introduce security hole in your system. Do NOT set this configuration with URL path other than application's healthcheck endpoint.
+	BypassURLPath string `yaml:"bypass_url_path"`
```

Comment:
should we support multiple bypass path?

Reply:
fixed: https://github.com/yahoojapan/authorization-proxy/pull/41#discussion_r372231063